lib/Zonemaster/CLI.pm

```diff
@@ -356,9 +356,13 @@ sub run {
     }
 
     # These two must come after any profile from command line has been loaded
-    # to override the profile setting, but only if selected.
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 ) if defined ($self->ipv4);
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 ) if defined ($self->ipv6);
+    # to make any IPv4/IPv6 option override the profile setting.
+    if defined ($self->ipv4) {
+        Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 );
+    }
+    if defined ($self->ipv6) {
+        Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 );
+    };
```

Comment:
unnecessary trailing semicolon

And I think that the `0+` can be removed as well.

Reply:
> And I think that the 0+ can be removed as well.

That's correct. It's in the contract of the Profile properties to always return the correct data type.

Reply:
Concerning "0+" I also looked at that, but I wanted to change as little as possible since this is a bug fix late in the cycle.

The tailing semicolon it not needed. When I started with Perl 1996 I think it was recommended, or maybe even mandatory in Perl 4, so I have a habit of writing them. It does not harm anything.

Reply:
I appreciate the historical note :)